flight/Modules/VtolPathFollower/vtol_follower_fsm.c

```diff
@@ -546,46 +507,22 @@ static int32_t do_land()
  */
 static int32_t do_loiter()
 {
-	const float LOITER_LEASH = 4;
-
-	LoiterCommandData loiterCommand;
-	LoiterCommandGet(&loiterCommand);
-
-	float yaw;
-	AttitudeActualYawGet(&yaw);
-	yaw *= DEG2RAD;
+	float att_adj[2] = { 0, 0 };
+	if (vtol_follower_control_loiter(DT, vtol_hold_position_ned, att_adj)) {
```

Comment:
here we should be using a temp variable instead of ```vtol_hold_position_ned```. a) this is for safety since that is one of the core control variables and probably should only be altered by ```hold_position``` called immediately below and b) to ensure when it doesn't do anything this variable isn't actually changed.